src/Components/CreateImageWizardV2/steps/Review/ReviewStep.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+
+import { ExpandableSection, Form, Title } from '@patternfly/react-core';
+
+import { ImageOutputList } from './imageOutput';
+
+import { Distributions } from '../../../../store/imageBuilderApi';
+
+type ReviewStepPropTypes = {
+  release: Distributions;
+  arch: 'x86_64' | 'aarch64';
```

Comment:
An existing type can be used after it's imported.

```suggestion
  arch: ArchitectureItem['arch'];
```